wp-content/themes/chrisdinh-photography/resources/views/sections/footer.blade.php

```diff
@@ -1,3 +1,62 @@
-<footer class="content-info">
-  @php(dynamic_sidebar('sidebar-footer'))
+@php
+  // Set the footer classes based on whether it's the homepage or not
+  $footerClasses = is_front_page() ? 'absolute bottom-0 w-full' : 'relative';
+
+  // Simplify conditionals by assigning contact information values to variables
+  $phoneNumber = $contactInformation['contact__phone-number'] ?? null;
+  $instagramLink = $contactInformation['contact__instagram-link']['url'] ?? null;
+  $emailAddress = $contactInformation['contact__email-address'] ?? null;
+@endphp
```

Comment:
Can we perform this logic in App.php and instead return `$phoneNumber`, `$instagramLink`, and `$emailAddress` as individual variables instead of having to find the information from one main variable in the template? This will help keep the template file clean from logic. Same thing applies to `$footerClasses` although that one is not as important

Reply:
Moved logic to App.php in commit 88671b2e3e58f5488e00d0f3c204bd0a315c4ce9